src/tags/bool.ts

```diff
@@ -0,0 +1,129 @@
+import { Cluster } from '../cluster';
+import { BaseSubtag, BBTagContext, SubtagCall } from '../core/bbtag';
+import { SubtagType, parse, bbtagUtil, compare } from '../utils';
+
+const operators = [
+    '==',
+    '!=',
+    '>=',
+    '>',
+    '<=',
+    'startswith',
+    'endswith',
+    'includes',
+    'contains'
+];
+
+interface OperatorHandler {
+    [index: string]: (a: string, b: string) => boolean;
+}
+
+export class BoolSubtag extends BaseSubtag {
+    public operatorHandler: OperatorHandler;
+    public operators: string[];
+    public constructor(cluster: Cluster) {
+        super(cluster, {
+            name: 'bool',
+            category: SubtagType.COMPLEX,
+            definition: [
+                {
+                    args: ['arg1', 'evaluator', 'arg2'],
+                    description:
+                        'Evaluates `arg1` and `arg2` using the `evaluator` and returns `true` or `false`. ' +
+                        'Valid evaluators are `' +
+                        operators.join('`, `') +
+                        '`\n' +
+                        'The positions of `evaluator` and `arg1` can be swapped.',
+                    exampleCode: '{bool;5;<=;10}',
+                    exampleOut: 'true',
+                    execute: (ctx, args, subtag) =>
+                        this.runCondition(
```

Comment:
Can we try to keep these on 1 line. I think its fine in this situation if it ends up being a long line as there should be little to no logic in these, its just a redirection to an actual method below so keeping the definition section easy to scan over is more important I think

Reply:
Sure!